index.html

```diff
@@ -34,7 +34,7 @@
             </ul>
           </div>
           <div class="hamburger">
-            <img src="img/Icons/Union.png" alt="#" />
+            <img src="img/Icons/Union.png" alt="hamburger-menue" />
           </div>
         </nav>
       </header>
```

Comment:
- [ ] Hey, you've done great with the accessibility, however, you're not done yet. I'd be listing all the changes you need to make to better increase your accessibility. Kindly refer to each link to see them and proceed with the changes

- kindly, improve your page title to be more personal and descriptive [here](https://github.com/Bushido-brown/My-Portfolio2/blob/4e03f8ecc0e346b16933d3b9c9b1713e33160e68/index.html#L16)
- - Use this [chrome extetnsion](https://chrome.google.com/webstore/detail/wcag-color-contrast-check/plnahcmalebffmaghcpcmpaciebdhgdf?hl=en) to analyze and fix all your color contrast issue below:

1. [connect in your CSS file](https://github.com/Bushido-brown/My-Portfolio2/blob/4e03f8ecc0e346b16933d3b9c9b1713e33160e68/styles/style.css#L141-L145)
2. [languages in your CSS file](https://github.com/Bushido-brown/My-Portfolio2/blob/4e03f8ecc0e346b16933d3b9c9b1713e33160e68/styles/style.css#L234-L245)
3. [background color in your css file](https://github.com/Bushido-brown/My-Portfolio2/blob/4e03f8ecc0e346b16933d3b9c9b1713e33160e68/styles/style.css#L359-L369)

-  - Form section:
- Kindly [Improve your form accessiblity](https://github.com/Bushido-brown/My-Portfolio2/blob/4e03f8ecc0e346b16933d3b9c9b1713e33160e68/index.html#L240-L264)  by adding the label tag to your form input and textarea.  A form should have the label tag as the label tag point or specify a label for an <input>, <select>, <textarea> element of a form to improve accessibility for screen users. There are many ways to add the label tag. use this for yours

`<label for="username"><input type="text" name="username" id="username" maxlength="30" placeholder="username" required></label>`